src/TodoList.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class TodoList extends React.PureComponent {
+  state = {
+    todo: this.props.todo,
+    users: this.props.user,
+    isTodoCompleted: false,
+  }
+
+  handlerCheckBox = () => {
+    this.setState(({ isTodoCompleted }) => ({
+      isTodoCompleted: !isTodoCompleted,
+    }));
+  }
+
+  render() {
+    const executor = (this.state.users
+      .find(user => user.id === this.state.todo.userId)).name;
+
+    return (
+      <>
+        <div className="list__item">
+          <p className="list__item_task">{this.state.todo.title}</p>
+          <div className={this.state.isTodoCompleted
+            ? 'list__item_completed'
+            : 'list__item_completed-not'}
```

Comment:
you can use classnames lib in all your project

Reply:
fixed